src/adcm_client/packer/data/config_data.py

```diff
@@ -61,13 +62,21 @@ def _from_file(self, key, **kwargs):
 
     def _from_data(self, key, **kwargs):
         try:
-            value = [entry.get(key) for entry in self.data
-                     if (entry.get('type') == 'cluster'
-                         or entry.get('type') == 'provider'  # noqa: W503
-                         or entry.get('type') == 'host')]  # noqa: W503
+            value = [
+                entry.get(key)
+                for entry in self.data
+                if (
+                    entry.get('type') == 'cluster'
+                    or entry.get('type') == 'provider'  # noqa: W503
```

Comment:
```suggestion
                    or entry.get('type') == 'provider'
```

we disabled flake here and moved to black, so there is no need to inline flake commands.